src/config/logger.ts

```diff
@@ -87,9 +154,13 @@ const createLoggerOptions = (label: string): LoggerOptions => {
   return {
     level: 'debug',
     format: format.combine(
+      format.errors({ stack: true }),
       format.label({ label }),
       format.timestamp(),
-      isDev ? format.combine(format.colorize(), customFormat) : format.json(),
+      errorHunter(),
+      !isDev
+        ? format.combine(format.colorize(), errorPrinter(), customFormat)
+        : format.json({ replacer: jsonErrorReplacer }),
```

Comment:
nit: swap the order around
```
isDev ? format.json({ replacer: jsonErrorReplacer }) : format.combine(format.colorize(), errorPrinter(), customFormat)
```

Reply:
Whoops you totally caught an error here too, we should be logging json if it's not dev!! Remnant of testing and pushing code late at night....